process/commands/AbstractCommand.java

```diff
@@ -0,0 +1,50 @@
+/*

```

Comment:
It seems that this file is already under `/unitex/src/fr/umlv/unitex/process/commands/AbstractCommand.java` is there any reason to try to create a new copy under `process/commands/AbstractCommand.java`?